test/aave-v2/TestSupplyVault.t.sol

```diff
@@ -240,4 +240,35 @@ contract TestSupplyVault is TestSetupVaults {
         assertEq(balanceOnPool, 0, "onPool amount not zero");
         assertEq(balanceInP2P, 0, "inP2P amount not zero");
     }
+
+    function testPreviewMint() public {
+        uint256 amount = 1e5 ether;
+
+        uint256 balanceBefore1 = ERC20(dai).balanceOf(address(vaultSupplier1));
+        uint256 balanceBefore2 = ERC20(dai).balanceOf(address(vaultSupplier2));
+
+        // This should use predicted update indexes.
```

Comment:
Those comments are not super clear because indexes are on Morpho and not on the vaults. Perhaps it would be better to just say something like that: "preview should be equal to the minted amount".

Reply:
I guess I wrote the comment badly, as that was not my intended message. I will rewrite the comment to convey better what I am trying to point out in the test design.